ores/wsgi/routes/scores.py

```diff
@@ -113,19 +115,41 @@ def score_revision(context, model, rev_id):
             return responses.not_found("No models available for {0}"
                                        .format(context))
 
-        precache = "precache" in request.args
-
-        # If the model exists, score revisions with it and return the result
         if model not in score_processor[context]:
             return responses.not_found("Model '{0}' not available for {1}."
                                        .format(model, context))
+
+        params = {}
+        if request.method == "POST":
+            data = request.get_json()  # Assume POSTs are JSON.
```

Comment:
Seems like a lot of complication.  The cache can't have a hierarchy to it.  It's only one-dimensional.  Since JSON object keys must be strings, I'm not sure if we get any benefit from JSON here.  We'd be implementing JSON-soap.